docs/cli/usage/permission-boundary.md

```diff
@@ -0,0 +1,16 @@
+---
+title: IAM Role Permission Boundary
+description: Apply a Permission Boundary to all IAM Roles created by Amplify CLI.
+---
+
+To set the maximum permissions that can be granted to IAM Roles created by Amplify CLI, a [Permission Boundary](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_boundaries.html) can be configured for the project.
+
+Amplify CLI does not create an IAM Policy to use as a Permission Boundary. This must be configured outside of the project (typically as part of an AWS Org rule or other enterprise rule).
```

Comment:
Include steps for creating the Permission Boundary via the console, including an example permission boundary. This should also highlight how/where to find the ARN.